monitor/plugins/spark_mesos/plugin.py

```diff
@@ -19,7 +19,6 @@
 import json
 import paramiko
 import pytz
-import requests
```

Comment:
This is a unused import? If yes, please comment this modification in the description of this PR.

Reply:
Done